modules/apps/headless/headless-admin-site/headless-admin-site-impl/src/main/java/com/liferay/headless/admin/site/internal/resource/v1_0/SiteResourceImpl.java

```diff
@@ -251,7 +222,8 @@ public Site postSite(Site site) throws Exception {
 	}
 
 	@Override
-	public Site postSiteSiteInitializer(MultipartBody multipartBody)
+	public Site postSiteSiteInitializer(
+			String siteExternalReferenceCode, MultipartBody multipartBody)
```

Comment:
Should this required the ERC in the endpoint? Previously it did not and now it does not match the new POST method. If it should be required then it should be used in the implementation.